lib/components/molecules/Toast/Toast.style.js

```diff
@@ -0,0 +1,56 @@
+import { css } from 'styled-components';
+
+const ToastStyles = css`
+  position: fixed;
+  display: flex;
+  align-items: center;
+  min-height: 70px;
+  min-width: 300px;
+  padding: 0 var(--spacing-3);
+  .toast-loader {
+    position: absolute;
+    bottom: 0;
+    height: 7px;
+    left: 0;
+    width: 100%;
+    background-color: var(--white);
+    opacity: 0.4;
+    animation: dismiss-anim ${props => `${props.autoDismissTimeout / 1000}.5s`} ease-in-out;
+  }
+  ${props =>
+    props.placement === `top-right`
+      ? `top: 30px;right: 30px;animation: var(--animation-slide-left);`
+      : ``};
+  ${props => (props.variation === `info` ? `${props.theme.colors.CTA_INFO};` : ``)};
+  ${props => (props.variation === `warning` ? `${props.theme.colors.CTA_WARNING};` : ``)};
+  ${props => (props.variation === `success` ? `${props.theme.colors.CTA_SUCCESS};` : ``)};
+  ${props => (props.variation === `error` ? `${props.theme.colors.CTA_ERROR};` : ``)};
+  ${props => (props.inheritedStyles ? props.inheritedStyles : '')};
+  .toast-text {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    ${props => `${props.theme.typography.FONT_TOAST}`};
+  }
+  .toast-text img {
+    padding-right: 0.8em;
+  }
+  .toast-close {
```

Comment:
To be updated along with ICON

Reply:
Created a New Issue Task in reference to https://github.com/pagesource/atomic-react-components/issues/190

Reply:
Can you also move your SVG icons lib/static/images to styles/themes/base/icons?